src/event_loop.rs

```diff
@@ -377,6 +309,30 @@ impl<T> EventLoopWindowTarget<T> {
     pub fn listen_device_events(&self, allowed: DeviceEvents) {
         self.p.listen_device_events(allowed);
     }
+
+    /// Sets the [`ControlFlow`].
+    pub fn set_control_flow(&self, control_flow: ControlFlow) {
+        self.p.set_control_flow(control_flow)
+    }
+
+    /// Gets the current [`ControlFlow`].
+    pub fn control_flow(&self) -> ControlFlow {
+        self.p.control_flow()
+    }
+
+    /// Send a [`LoopExiting`] event and stop the event loop.
+    ///
+    /// [`LoopExiting`]: Event::LoopExiting
+    pub fn exit(&self) {
+        self.p.exit()
+    }
+
+    /// Returns if the [`EventLoop`] is about to stop.
+    ///
+    /// See [`exit()`](Self::exit).
+    pub fn is_exit(&self) -> bool {
+        self.p.is_exit()
+    }
```

Comment:
`is_exit` is a pretty bad name for internal state (event loop is not a door in the end to be marked as exit). `exiting` should be better, since it reflects the `state` not what it is.

It's also a bit complicated when it comes to deref impl on the `EventLoop`, since you can sort of exit it (in case of ondemand/pump_events), but you can rerun it. Do we really need the getter for the exit status in the end?

Reply:
I changed it to `exiting()`.

> It's also a bit complicated when it comes to deref impl on the `EventLoop`, since you can sort of exit it (in case of ondemand/pump_events), but you can rerun it. Do we really need the getter for the exit status in the end?

`exiting()` returns false after the `EventLoop` is reset, so shouldn't that be fine?

Reply:
I guess.